training.md

```diff
@@ -0,0 +1,34 @@
+---
+layout: training
+title: Training Materials
+---
+
+# Foreman training materials
+
+This page lists the materials available for those wishing to study/practice working
+with Foreman, as an alternative style to reading the manual. 
+
+## Netways Foreman Training Course
+        
+This course was developed by Netways and made available to the Foreman
+community for use in developing training courses of their own, or for community
+members to use for self-study. It is designed as a two-day hands-on training
+course, and as well as the presentation below, it comes with
+[handouts][handouts], [exercises][exercises], and [solutions][solutions].
+
+[handouts]: https://github.com/NETWAYS/foreman-training/releases/download/v1.0/foreman-training-handouts.pdf
+[exercises]: https://github.com/NETWAYS/foreman-training/releases/download/v1.0/foreman-training-exercises.pdf
+[solutions]: https://github.com/NETWAYS/foreman-training/releases/download/v1.0/foreman-training-solutions.pdf
+
+A full-page version of the presentation can be found at
+[netways.github.io/foreman-training/static][rendered]
+
+[rendered]: //netways.github.io/foreman-training/static
+
+The source for this training material, along with handouts and
```

Comment:
I'd say move this paragraph about the source below the presentation itself. I think the first thing people will look for is the full-page presentation, as the rendered one is very small right now. 

If you move this paragraph below the rendered presentation, then the link to the  full-page thing is right above the presentation and it's easy to find (https://github.com/theforeman/theforeman.org/pull/605/files#diff-6493d41e298135faa05bc270a80fc0e4R23). Alternatively you can make the iframe clickable so that it opens a new tab with the presentation in netways.github.io